src/tests/multihost/alltests/test_automount.py

```diff
@@ -342,3 +342,83 @@ def test_008_wildcardsearch(self, multihost, indirect_nismaps,
         # delete the pcap file
         del_pcap = 'rm -f %s' % auto_pcapfile
         multihost.client[0].run_command(del_pcap)
+
+    @pytest.mark.parametrize('add_nisobject', ['/export'], indirect=True)
+    @pytest.mark.tier1
+    def test_009_fetch_maps_coming_online_from_offline(self, multihost,
+                                                       add_nisobject):
+        """
+        @Title: fetch autofs map after coming online from offline
+
+        @ID: b9da6e0e-3d8b-4465-b435-338708d0d51e
+
+        @Bugzilla:
+        https://bugzilla.redhat.com/show_bug.cgi?id=1113639
+
+        @Setup:
+        1. edit sssd.conf and specify autofs_provider
+        2. Install and configure firewall
+
+        @Steps:
+        1. firewall block 389 and 636
+        2. stop sssd, autofs.
+        3. remove sssd cache
+        4. Start sssd
+        5. remove firewall rule
+        6. start autofs
+
+        @Expectedresults:
+        1. sssd should come online within 60 seconds
+        2. autofs should start successfully
+        """
+        multihost.master[0].run_command(['touch', '/export/nfs-test'])
+        for service in ['sssd', 'autofs']:
+            srv = 'systemctl stop %s' % service
+            try:
+                multihost.client[0].run_command(srv)
+            except subprocess.CalledProcessError:
+                pytest.fail("Unable to stop %s service" % service)
```

Comment:
Please update the code with service_ctrl method from #5512 here and other places below